pytorch_lightning/accelerators/accelerator.py

```diff
@@ -248,26 +263,36 @@ def setup_training_type_plugin(self) -> None:
 
     def setup_precision_plugin(self) -> None:
         """Attaches the precision plugin to the accelerator."""
-        model, optimizers, schedulers = self.precision_plugin.connect(self.model, self.optimizers, self.lr_schedulers)
+        model, optimizers, schedulers = self.training_type_plugin.precision_plugin.connect(
+            self.model, self.optimizers, self.lr_schedulers
+        )
         self.model = model
         self.optimizers = optimizers
         self.lr_schedulers = schedulers
 
     @property
     def amp_backend(self) -> Optional[LightningEnum]:
-        if isinstance(self.precision_plugin, ApexMixedPrecisionPlugin):
+        if isinstance(self.training_type_plugin.precision_plugin, ApexMixedPrecisionPlugin):
             return AMPType.APEX
-        if isinstance(self.precision_plugin, NativeMixedPrecisionPlugin):
+        if isinstance(self.training_type_plugin.precision_plugin, NativeMixedPrecisionPlugin):
             return AMPType.NATIVE
         return None
 
     @property
     def precision(self) -> Union[str, int]:
-        return self.precision_plugin.precision
+        """This method is deprecated and will be removed soon.
+
+        Use `training_type_plugin.precision_plugin.precision` instead.
+        """
+        rank_zero_deprecation(
+            f"`{self.__class__.__name__}.precision` was and will be removed soon"
+            f" Use `training_type_plugin.precision_plugin.precision` instead."
+        )
```

Comment:
Can you merge/rebase master into this PR? To avoid any potential failures after #9940

Reply:
Checked